docs/.vitepress/config.mts

```diff
@@ -296,6 +283,44 @@ function sideNavDocs() {
             },
           ],
         },
+        {
+          text: 'Vue',
```

Comment:
Why Vue back to unstable?

Reply:
I was unsure if you consider it stable or alpha. In the readme of @nxext/vue, it's written that it's unstable. I just wanted to make things consistent. If we consider @nxext/vue stable, I can move it back and update the package readme to remove the unstable warnings.

Reply:
Oh I just forgot to remove that from the README

Reply:
Okay, cool :-) Let me move it back and remove the warnings from the README